store/mongo/datastore_mongo.go

```diff
@@ -431,8 +433,48 @@ func (db *DataStoreMongo) SaveToken(ctx context.Context, token *jwt.Token) error
 	} else if err != nil {
 		return errors.Wrap(err, "failed to store token")
 	}
+	return err
+}
 
-	return nil
+func (db *DataStoreMongo) EnsureSessionTokensLimit(ctx context.Context, userID oid.ObjectID,
+	tokensLimit int) error {
+	opts := &mopts.FindOptions{}
+	opts.SetLimit(int64(tokensLimit))
+	opts.SetSkip(int64(tokensLimit) - 1)
+	opts.SetSort(bson.M{
+		DbTokenIssuedAtTime: -1,
```

Comment:
ok I checked this one (for off-by one which is always killing me). we have the newest token first, we skip `tokensLimit-1` and then we remove all that are older than the 10th one.